docs/getting-started.md

```diff
@@ -1,8 +1,262 @@
+
+```c++
+std::vector<double> rho(mix.nSpecies()), T(mix.nEnergyEqns());
+mix.getTemperatures(&T[0]);
+mix.densities(&rho[0]);
+```
+
+Now, we let's explicitly set the state, using species densities and mixture temperatures.  For the default state model used in the `air_5` example, this corresponds to an index of 1.
+
+```c++
+mix.setState(&rho[0], &T[0], 1);
+```
+
+Printing the mixture state as before, we see that is unchanged, as expected.
+
+```
+Temperature(s) [K]: 5000 
+Pressure [Pa]: 101325
+Species mole fractions:
+N: 0.0269021
+O: 0.324738
+NO: 0.0174998
+N2: 0.628901
+O2: 0.00195878
+```
+
+While we used the current state to demonstrate the `setState` call here, typically the state information will be known.  For example, this could be provided by an external CFD tool which is coupled to Mutation++.
+
+### Computing physicochemical properties
+
+Once we have set the state of the mixture, we can proceed to compute a variety of thermochemical properties of the gas.  For example, here are few properties computed with the state set in the previous section.
+
+**Code**
+```c++
+std::cout << "\nMixture enthalpy [J/kg]: " << mix.mixtureHMass();
+std::cout << "\nMixture entropy [J/kg-K]: " << mix.mixtureSMass();
+
+std::vector<double> lambda(mix.nEnergyEqns());
+mix.frozenThermalConductivityVector(&lambda[0]);
+std::cout << "\nMixture thermal conductivities [W/m-K]: ";
+for (auto v: lambda) std::cout << v << ' ';
+
+std::cout << "\nMixture dynamic viscosity [Pa-s]: " << mix.viscosity();
+```
+
+**Output**
+```
+Mixture enthalpy [J/kg]: 9.9924e+06
+Mixture entropy [J/kg-K]: 11335.9
+Mixture thermal conductivities [W/m-K]: 0.280127 
+Mixture dynamic viscosity [Pa-s]: 0.000143242
+```
+
+Of course, there are many more properties implemented in Mutation++.  See the API documentation for more details.
+
+## Specifying non-default mixture options
+
+In the above examples, we loaded the mixture using only the mixture name.  This name refers to a mixture file in the data directory provided with the library.  Sometimes, it is necessary to modify the default options listed in the mixture file.  This can be done using the `MixtureOptions` class. 
+
+```c++
+MixtureOptions opts("air_5");
+opts.setStateModel("ChemNonEqTTv");
+opts.setThermodynamicDatabase("RRHO");
+opts.setViscosityAlgorithm("Gupta-Yos");
+Mixture mix(opts);
+```
+
+Note that we can instantiate a mixture options object using the default options in a mixture file.  You can also create a MixtureOptions object with no defaults, but you must specify every option manually in this case.  In the above, we have used the defaults for the `air_5` mixture, but explicitly set the state model, thermodynamic database, and viscosity algorithm.  For example, we have replaced the 1-temperature default model with the 2-temperature nonequilibrium model.  Printing the mixture properties with the new options yields the following output.
```

Comment:
Maybe a link to all the options in the input file would help.

Reply:
done